pkg/client/saramaclient.go

```diff
@@ -210,3 +210,60 @@ func (s *SaramaClient) GetConfig(resource ConfigResource) ([]ConfigEntry, error)
 
 	return configEntries, nil
 }
+
+func (s *SaramaClient) GetEmptyTopics() ([]string, error) {
+	brokerMap := s.ListBrokers()
+	brokerIds := make([]int32, 0)
+	for id := range brokerMap {
+		brokerIds = append(brokerIds, int32(id))
+	}
+	metaData, err := s.admin.DescribeLogDirs(brokerIds)
+	if err != nil {
+		return nil, err
+	}
+	topicWiseMap, err := getTopicWiseMetaDataMap(metaData, brokerMap)
```

Comment:
we don't need this transformation to filter zero size, we can directly filter on `DescribeLogDirsResponseDirMetadata` map